src/main/proto/service.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+// Feature Extractor Service
+service FeatureExtractor {
+  // Extract identifiers weighted set
+  rpc Identifiers (IdentifiersRequest) returns (FeaturesReply) {}
+  // Extract literals weighted set
+  rpc Literals (LiteralsRequest) returns (FeaturesReply) {}
+  // Extract uast2seq weighted set
+  rpc Uast2seq (Uast2seqRequest) returns (FeaturesReply) {}
+}
+
+// The identifiers request message containing extractor configuration and uast.
+message IdentifiersRequest {
+  bytes uast = 1;
```

Comment:
One thing that we discussed was using a super-set of our existing UAST msg type.

This can be done by adding adding 2 files to our `/proto/` dir 
 - https://github.com/bblfsh/client-scala/blob/master/src/main/proto/github.com/gogo/protobuf/gogoproto/gogo.proto
 - https://github.com/bblfsh/sdk/blob/master/uast/generated.proto

and then

```
import "github.com/gogo/protobuf/gogoproto/gogo.proto";
import "gopkg.in/bblfsh/sdk.v1/uast/generated.proto";
```

and 

```
uast gopkg.in.bblfsh.sdk.v1.uast.generated.Node = 1;
```

Reply:
yes. sorry.